src/asmcnc/comms/router_machine.py

```diff
@@ -1428,6 +1428,10 @@ def g28_z(self): return float(self.s.g28_z)
 # SPEED AND FEED GETTERS
     def feed_rate(self): return int(self.s.feed_rate)
 
+    def get_is_constant_feed_rate(self, last_modal_feed_rate, feed_override_percentage, current_feed_rate):
```

Comment:
this function isn't very general, but it can live here

Reply:
Even though it's only used by YP at the moment, it's not dependent on other parts of the YP algorithm, and I think it could have uses outside of YP in future if we wanted :)